models/tridimensional/docking_validation/batch_dock.sh

```diff
@@ -74,16 +87,25 @@ if [ ! -z $set_partners ]; then
     echo "Adding '--set-partners $set_partners' argument to dock_variants"
     opt_dock_variants_args+=" --set_partners $set_partners"
 fi
+if [ ! -z $pam64 ]; then
+    echo "Passing '--pam64' argument to dock_variants"
+    opt_dock_variants_args+=" --pam64"
+    pam_end=63
+fi
+fi p ! -z $pam_tool]; then
```

Comment:
yeah, this should be `if [ ! -z $pam_tool ];`, otherwise seemed to work when I ran on server